presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -309,7 +309,9 @@ protected static ResultSet getColumns(JdbcTableHandle tableHandle, DatabaseMetaD
             return Optional.of(ColumnMapping.sliceMapping(
                     createUnboundedVarcharType(),
                     varcharReadFunction(),
-                    varcharWriteFunction(),
+                    (statement, index, value) -> {
+                        throw new PrestoException(NOT_SUPPORTED, "Underlying type is not supported for INSERT: " + typeHandle.getJdbcTypeName().orElse("?"));
```

Comment:
`typeHandle.getJdbcTypeName().get()` + message saying that type is "forced mapped to varchar"

and `// TODO this should be handled during planning phase`